test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -2752,6 +2889,19 @@ static int crypto_init(odp_instance_t *inst)
 		return -1;
 	}
 
+	if (odp_schedule_capability(&sched_capa)) {
+		fprintf(stderr, "odp_schedule_capability() failed\n");
+		return -1;
+	}
+
+	odp_schedule_config_init(&sched_config);
+
+	/* Configure the scheduler. All test cases share the config. */
+	if (odp_schedule_config(&sched_config)) {
```

Comment:
odp_schedule_config(NULL) could be used instead when all config params have default values.

Reply:
Agreed.